### ISSUE TITLE
Additional issues.

### ISSUE THREAD

**Issue by muschellij2** [muschellij2/flexconn#4]
- [ ] Add test/train split
- [ ] Allow thresholding determination
- [ ] Add thresholding into prediction

**Comment by skeydan** [comment]
added train-test split to ```flexconn_wpackage.R```

Don't know if you want to put that in a function?

**Comment by muschellij2** [comment]
I don't think so as `keras::fit` has the `validation_split` argument to do this specifically.  If the data is shuffled (which is why they did it), then this will do a split on random data.

**Comment by skeydan** [comment]
That's for a different purpose. It's a 3-way split (or I would want it to be, if I was a user!) - training set, validation set, test set.
The validation set gets used during training, so we still need something that's being left alone.

**Comment by muschellij2** [comment]
I think these processes are more of a `script` thing than a `function`/`package` thing.  Do you agree?

**Comment by skeydan** [comment]
yes, totally - I just put it there as something TBD :-)

**Comment by muschellij2** [comment]
OK - you need to reinstall flexconn (and probably do predictions again).

Large typo
https://github.com/muschellij2/flexconn/commit/82007d12a43b01b57a2a26c23ada8bc7400e6108#diff-51cfef91a9b53f8aa2622e82a59a1a31R162
that copied the T1 image to the flair and never used the FLAIR.

Also, I used the percentiles from numpy so we should be getting almost
identical normalizations.  For some reason, however, I'm not getting the
exact predictions as FLEXCONN is, even using the same model (the hd5 file
they include with their package).

Need to look into further.
John

**Comment by skeydan** [comment]
Yeah, thanks, I've run git pull and I'll check now how it works for me :-)

**Comment by skeydan** [comment]
>>For some reason, however, I'm not getting the exact predictions as FLEXCONN is, even using the same model (the hd5 file they include with their package).

how big are the differences ("how big" is vague I know, but in some measure...)?

**Comment by muschellij2** [comment]
Very small (like correlation is 0.999994), but a few voxels get classified
differently with a threshold of 0.34 for example.
John
